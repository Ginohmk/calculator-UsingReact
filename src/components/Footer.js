import React from 'react';

export default function Footer() {
  return (
    <footer>
      <p data-testid="footer-data">footer</p>
    </footer>
  );
}
