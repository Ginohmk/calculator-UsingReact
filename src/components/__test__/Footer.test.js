import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../Footer';

describe('TEst for Home component', () => {
  test('Home snapShot test', () => {
    const tree = renderer.create(<Footer />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
