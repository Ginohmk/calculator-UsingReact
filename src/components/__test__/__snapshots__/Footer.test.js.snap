// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TEst for Home component Home snapShot test 1`] = `
<footer>
  <p>
    footer
  </p>
</footer>
`;
